internal/integration/api/reset.go

```diff
@@ -139,7 +140,8 @@ func (suite *ResetSuite) TestResetWithSpecState() {
 
 	userDisksToWipe := xslices.Map(
 		xslices.Filter(disks.Messages[0].Disks, func(disk *storage.Disk) bool {
-			return !disk.SystemDisk
+			// The DVD-ROM check should be removed for something more reliable.
+			return !disk.SystemDisk && !disk.Readonly && !strings.Contains(disk.Model, "DVD-ROM")
```

Comment:
we can add a specific field that it's a CD, plus it should report a small size, I can look into that

Reply:
@smira yeah, that would be good